extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -100,6 +103,19 @@ protected function renderItem( $type, array $data ) {
 		return $result;
 	}
 
+	protected function renderData( $nodeData ) {
+		if (
+			!empty( $nodeData['collapse'] ) &&
+			$nodeData['collapse'] === \Wikia\PortableInfobox\Parser\Nodes\NodeData::COLLAPSE_CLOSED_OPTION &&
```

Comment:
I'm wondering if we should couple those two classes. They are in a separate layers, and I'd like to keep it that way.